harpy/_validations.py

```
@@ -10,6 +10,7 @@
 from rich import print as rprint
 from rich.table import Table
 import rich_click as click
+from ._parsers import contigs_from_vcf
```

Ruff (0.8.2)

`._parsers.contigs_from_vcf` imported but unused

Remove unused import: `._parsers.contigs_from_vcf`

(F401)